packages/trace-viewer/src/ui/uiModeView.css

```diff
@@ -91,11 +91,17 @@
   flex-direction: row;
   align-items: center;
   height: 30px;
+  overflow: hidden;
+  gap: 6px;
```

Comment:
I would stick with a 8px grid and use `8px` here. We lose some space but it feels much nicer to me.